### PR TITLE
LPS-39084

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java [hunk 1]
```diff
@@ -751,9 +771,35 @@ private boolean _containsPortlet(Layout layout, String portletId) {
...
+			clusterRequest.setFireAndForget(true);
+
+			ClusterExecutorUtil.execute(clusterRequest);
+		}
+		catch (Throwable throwable) {
+			_log.error("Unable to notify slave", throwable);
+		}
+	}
+
+	private void _updateLogLevels(List<LogLevel> logLevels) {
```
Comment (tinatian) [inline modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java]: @marianoalvarosaiz Did you test your change locally? I believe the code is not working correctly. This _updateLogLevels method should be static as when we invoke the method, it is doing methodHandler.invoke()

**Comment by tinatian** [comment]
@vicnate5 Could you please add test to cover this?

Thanks.
Tina.

**Comment by vicnate5** [comment]
@marianoalvarosaiz I sent you functional test coverage to add to this branch https://github.com/marianoalvarosaiz/liferay-portal/pull/658.

**Comment by marianoalvarosaiz** [comment]
Merged @vicnate5, thanks!

**Comment by vicnate5** [comment]
ci:test:clustering

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java (hunk 1 above)
Comment (marianoalvarosaiz) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java]: @tinatian, I don't fully understand your concern about having to be static. I tested locally and it works. Also I've passed Victor's test and it works, see https://github.com/marianoalvarosaiz/liferay-portal/pull/661 and https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2/22195/

**Comment by vicnate5** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
Cluster test results are good, functional test for this issue is passing. Now running relevant.

**Comment by vicnate5** [comment]
@tinatian This looks good

**Comment by marianoalvarosaiz** [comment]
Hi @tinatian,
 do you have any updates?
Kind regards.

**Comment by tinatian** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/125

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**